packages/vm/src/imports.rs

```diff
@@ -362,43 +352,11 @@ pub fn do_ed25519_batch_verify<A: BackendApi, S: Storage, Q: Querier>(
 }
 
 pub fn do_sha1_calculate<A: BackendApi, S: Storage, Q: Querier>(
-    env: &Environment<A, S, Q>,
-    hash_inputs_ptr: u32,
+    _env: &Environment<A, S, Q>,
+    _hash_inputs_ptr: u32,
 ) -> VmResult<u64> {
```

Comment:
Is it necessary change it to `VmResult<()>`

Reply:
It needs to be `VmResult<u64>`. u64 indicates the returned status of a call and our VM should return 10 which specifies a generic error.